packs/vsphere/actions/get_vm.json

```diff
@@ -0,0 +1,69 @@
+{
+    "name": "get_vm",
+    "runner_type": "run-python",
+    "description": "Retrieves the virtual machines on a vCenter Server system. It computes the union of Virtual Machine sets based on each parameter.",
```

Comment:
get_vms reflects the meaning better. Also, all the parameters, since they accept coma-separated values, would make sense to make plural.
Do you want to go this route?